features/tags_and_wrangling/tag_wrangling.feature

```diff
@@ -189,6 +189,10 @@ Feature: Tag wrangling
     Then I should see "Tag was updated"
       And I should see "Stargate Atlantis"
 
+    # A wrangler should see the reindex button
```

Comment:
This test scenario is not the greatest the way it's currently written -- they're really only supposed to test one thing (lolsob) -- so if you have a minute to make at least this part of it its own scenario, like:

```
Scenario: Wrangler has option to reindex a work

Given the work "Indexing Issues"
  And I am logged in as a tag wrangler
When I view the work "Indexing Issues"
Then I should see "Reindex Work"
```

that'd be dandy. If not, it's an existing problem, with this particular test, so it's not worth holding up the code for it.